sources/networkmanager/site_test.go

```diff
@@ -0,0 +1,74 @@
+package networkmanager
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/service/networkmanager"
+	"github.com/aws/aws-sdk-go-v2/service/networkmanager/types"
+	"github.com/overmindtech/aws-source/sources"
+	"github.com/overmindtech/sdp-go"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func (t *TestClient) GetSites(ctx context.Context, params *networkmanager.GetSitesInput, optFns ...func(*networkmanager.Options)) (*networkmanager.GetSitesOutput, error) {
+	return &networkmanager.GetSitesOutput{
+		Sites: []types.Site{
+			{
+				Tags:            []types.Tag{},
+				SiteId:          sources.PtrString("site1"),
+				GlobalNetworkId: sources.PtrString("default"),
+			},
+			{
+				Tags:            []types.Tag{},
+				SiteId:          sources.PtrString("site2"),
+				GlobalNetworkId: sources.PtrString("other"),
+			},
+		},
+	}, nil
+}
+
+func TestSiteOutputMapper(t *testing.T) {
+	output := networkmanager.GetSitesOutput{
+		Sites: []types.Site{
+			{
+				SiteId:          sources.PtrString("site1"),
+				GlobalNetworkId: sources.PtrString("default"),
+			},
+		},
+	}
+	scope := "123456789012.eu-west-2"
+	items, err := siteOutputMapper(context.Background(), &TestClient{}, scope, &networkmanager.GetSitesInput{}, &output)
+
+	if err != nil {
+		t.Error(err)
+	}
+
+	for _, item := range items {
+		if err := item.Validate(); err != nil {
+			t.Error(err)
+		}
+	}
+
+	if len(items) != 1 {
+		t.Fatalf("expected 1 item, got %v", len(items))
+	}
+
+	item := items[0]
+
+	// Ensure unique attribute
+	require.NotNil(t, item.Attributes)
+	uniqueAttr, err := item.Attributes.Get("globalNetworkSiteId")
+	require.Nil(t, err)
+	require.Equal(t, "default/site1", uniqueAttr.(string))
```

Comment:
I'd suggest that we don't use testify here just for consistency as it's not used anywhere else. Also we can simplify this a bit using the existing `.Validate()` method which checks for things like the unique attribute existing, so we could do something like:

```go
err := item.Validate()
if err != nil {
  t.Error(err)
}
if item.UniqueAttributeValue() != "default/site1" {
  t.Error("error goes here")
}
```